mastracode/web/.env.schema

```
@@ -28,6 +28,18 @@ MASTRACODE_PUBLIC_URL=
+# Slack request signing secret (Basic Information → App Credentials). This is
+# the switch for Slack as a whole: unset → no Slack integration is constructed
+# and the Connections page reports Slack as not set up. Also the last-resort
+# state-signer secret (see GITHUB_APP_WEBHOOK_SECRET below), so a Slack-only
+# deployment has a signer that survives a restart.
+# @sensitive
+SLACK_APP_SIGNING_SECRET=
+# Slack bot token (OAuth & Permissions → Bot User OAuth Token, starts xoxb-).
+# Lets the app post messages and open DMs; without it Slack can reach the app
+# but the app cannot answer.
+# @sensitive
+SLACK_APP_BOT_TOKEN=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SLACK_APP_BOT_TOKEN key should go before the SLACK_APP_SIGNING_SECRET key

(UnorderedKey)